### DISCUSSION TITLE
[keyball44] 右手 (トラックボール) 側が反応しなくなる

### DISCUSSION THREAD

**Discussion by satoshi5454** [Yowkees/keyball#486] [Q&A]
お世話になります。
先日は keyball46 の質問に丁寧に対応いただき、ありがとうございました。
トラックボール付きが大変使いやすく、新たに keyball44 も購入させていただきました。

その keyball44 なのですが、問題ないときは非常に快適なのですが、
右手 (トラックボール) 側が反応しなくなることがあります。
具体的な症状は以下の 2 点となります。

1. 右手側 親指の外側キー (写真の赤丸のキー) が反応しない。
　問題なく使えることもあるのですが、体感として 9 割は反応しません。
　ロープロファイル側を使用しています。

2. 右手側が、トラックボールを含め、何も反応しない。
　使用していると、突然反応しなくなります。
　一度 USB ケーブルを外し、TRRS ケーブルも差し直し、再度 USB ケーブルをつなぐと、
　また正常に認識されます。

半田付け面の写真も添付いたします。
修正方法等、ご教示いただけますと幸いです。

よろしくお願いいたします。

![750F17F3-D34D-4E0E-8114-2A4B91FCC0BD_1_105_c](https://github.com/Yowkees/keyball/assets/938874/f700ab1c-df9e-4216-a569-d3772d18bcbd)
![523FF1A5-4254-4E95-89E6-8EF43E987B67_1_105_c](https://github.com/Yowkees/keyball/assets/938874/5e529e9a-1d8b-4a8e-9f74-f9fbbf9fef1f)

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

Keyball46に続いてKeyball44もご購入いただきまして誠にありがとうございます。

添付しました写真の赤い丸で示した端子のはんだが隣の端子に触れているように見えますがご確認いただけますでしょうか。
隣の端子に触れている場合は少しはんだの量を減らしていただければと思います。

反応のない親指キーにつきまして、写真で見るかぎりははんだ付けにも問題ないように見えますがもう一度はんだ付けをやり直していただければと思います。反応が無いキーに対応するダイオードは緑の丸で示したものになります。
![305725869-f700ab1c-df9e-4216-a569-d3772d18bcbd](https://github.com/Yowkees/keyball/assets/143698349/78df2bd5-d8ac-4335-a952-57acda37e3f2)

**Comment by satoshi5454** [comment]
早々のご連絡、ありがとうございます！
ご指摘いただきました 3 点を修正しましたところ、正常に動作しています！
めちゃめちゃ快適です！
ありがとうございます！！

**Comment by Ryo624** [comment]
正常に動作するようになり良かったです！ご対応ありがとうございます。
ぜひ末永くKeyball46、44をご愛用いただければ幸いです！
